cmd/pbackend/api.go

```diff
@@ -63,6 +45,11 @@ func main() {
 	}
 	defer config.StopFeatureFlags(ctx)
 
+	// set GOMAXPROCs for Kubernetes
+	_, _ = maxprocs.Set(maxprocs.Logger(func(format string, args ...interface{}) {
```

Comment:
Wasn't this introduced mainly for the worker? should we do it there as well? :thinking:

Reply:
Oh no this tiny library performs some heuristics "guess" of what is the best goroutines max setting for HTTP handling in kubernetes. So it is actually only for API, for workers we don’t have that. At least the documentation states this is for HTTP traffic.

For other pods we use the default value which is I think 3 parallel goroutines (2 cores + 1).

Reply:
Ah! Cool, makes sense 👍